doc/source/io.rst

```diff
@@ -4056,31 +4056,59 @@ Compression
 +++++++++++
 
 ``PyTables`` allows the stored data to be compressed. This applies to
-all kinds of stores, not just tables.
+all kinds of stores, not just tables. Two parameters are used to
+control compression: ``complevel`` and ``complib``.
+
+``complevel`` specifies if and how hard data is to be compressed.
```

Comment:
I think let's change the defaults to ``complevel=None``, ``complib=None``.

If then complevel is not None and > 0 you can set complib to zlib (if it not defined)
if complib is not None and complevel is not 0 then you set the filter.